mlflow/server/js/src/common/components/EditableTagsTableView.js

```diff
@@ -92,6 +92,8 @@ export class EditableTagsTableViewImpl extends Component {
                     description:
                       'Error message for name requirement in editable tags table view in MLflow',
                   }),
+                },
+                {
```

Comment:
Is this change necessary?

Reply:
yes, error msg is not appropriate when tag name already exists.

<img width="599" alt="Screenshot 2023-02-14 at 4 18 19 PM" src="https://user-images.githubusercontent.com/1637727/218892875-10946d40-1f4a-4ffb-b8ad-1f047aae410c.png">

Reply:
Got it - thank you!